src/app/product-dialog/product-dialog.component.html

```diff
@@ -0,0 +1,130 @@
+<div class="mat-container">
```

Comment:
Nothing Specific. but as a reminder we still need another iteration here to add [(ngModel)]
NOT MERGE BLOCKING

Reply:
I had modified it and did not find the code after I did the pull request! so I modified it again